### ISSUE TITLE
Intent to implement: Vine simple embed

### ISSUE THREAD

**Issue by niallkennedy** [ampproject/amphtml#1005]
A [Vine simple embed](https://vine.co/v/MdKjXez002d/embed) is a single video file with an equal width and height loaded through an `iframe`.

![vine-simple-embed](https://cloud.githubusercontent.com/assets/1100/11375785/1064c0be-9292-11e5-82d7-7b6ad9f3014f.jpg)

A Vine video URL consists of a base URL and a Vine video identifier. Example: `https://vine.co/v/MdKjXez002d`

In the example above the Vine video identifier is `MdKjXez002d`. A Vine video identifier is required to display a video. A webpage URL suitable to include as framed content may be created from the extracted identifier.
```javascript
iframe.src = 'https://vine.co/v/' + vineid + '/embed/simple'
```

Videos are expected to be rendered with an equal width and height.

Vine supports a `pause` `postMessage` event to pause video. If the current document becomes inactive the Vine video is expected to pause.

**Comment by cramforce** [comment]
LGTM

**Comment by erwinmombay** [comment]
closed via #1006
